partitura/io/importmidi.py

```diff
@@ -218,7 +217,7 @@ def load_performance_midi(
 
         # add note id to every note
         for k, note in enumerate(notes):
-            note["id"] = f"n{k}"
+            note["id"] = f"n{k+1}"
```

Comment:
Is this necessary for something? It breaks every single alignment file in (n)ASAP and the link between performance midis and match files everywhere else?